compiler/gentree.cpp

```diff
@@ -1024,10 +1042,34 @@ VertexAdaptor<op_return> GenTree::get_return() {
     }
     ret = VertexAdaptor<op_return>::create(return_val);
   }
+
   CE (expect(tok_semicolon, "';'"));
   return ret.set_location(location);
 }
 
+VertexAdaptor<op_yield> GenTree::get_yield() {
+  auto location = auto_location();
+  next_cur();
+  skip_phpdoc_tokens();
```

Comment:
Why we should skip phpdoc here?

Reply:
Yes, that's an extra action. I'll delete it.